Userland/Libraries/LibJS/Runtime/Temporal/Duration.cpp

```diff
@@ -409,15 +434,15 @@ Crypto::SignedBigInteger total_duration_nanoseconds(double days, double hours, d
         // a. Set nanoseconds to nanoseconds - offsetShift.
         result_nanoseconds = result_nanoseconds.minus(Crypto::SignedBigInteger::create_from(offset_shift));
     }
-    // 3. Set hours to ℝ(hours) + ℝ(days) × 24.
+    // 3. Set hours to hours + days × 24.
```

Comment:
You're missing a numbering shift here, this is now step 2

Reply:
Thanks, fixed!